channels/stable

```diff
@@ -55,6 +55,9 @@ spec:
     networking:
       kubenet: {}
   kubernetesVersions:
+  - range: ">=1.18.0-alpha.1
+    recommendedVersion: 1.18.0-beta.1
+    requiredVersion: 1.18.0-alpha.1
```

Comment:
should these be:
```suggestion
    recommendedVersion: 1.18.3
    requiredVersion: 1.18.0
```
?

Reply:
Doh, thank you

Reply:
Fixed thanks. Too much on the clipboard.